editor/cypress/e2e/createManually.cy.js

```diff
@@ -0,0 +1,38 @@
+/// <reference types="cypress" />
+
+import 'cypress-file-upload';
+import 'cypress-iframe';
+
+
+describe('Create a resource manually', () => {
+  it('should allow adding a FileObject resource', () => {
+    // Streamlit starts on :8501.
+    cy.visit('http://localhost:8501')
+    cy.get('button', {timeout: 10000}).contains('Create', {timeout: 10000}).click()
+    cy.get('input[aria-label="Name:red[*]"]').type('MyDataset').blur()
+    cy.get('[data-testid="stMarkdownContainer"]')
+    .contains('Metadata')
+    .click()
+    cy.get('input[aria-label="URL:red[*]"]').type('https://mydataset.com', {force: true})
+    
+    // Create a resource manually.
+    cy.get('[data-testid="stMarkdownContainer"]').contains('Resources').click()
+    cy.get('[data-testid="stMarkdownContainer"]').contains('Add manually').click()
+
+    cy.get('input[aria-label="File name:red[*]"]').type('test.csv').blur()
+    cy.get('input[aria-label="SHA256"]').type('abcdefgh1234567').blur()
+    cy.get('button').contains('Upload').click()
+
+    // The file is created, so we can click on it to see the details.
+    cy.enter('[title="components.tree.tree_component"]').then(getBody => {
+      getBody().contains('test.csv').click()
+    })
+    
+    // cy.contains('input[aria-label="Name:red[*]"][value="test.csv"]').scrollIntoView();
+    // cy.get('input[aria-label="Name:red[*]"][value="test.csv"]')
```

Comment:
Could it have to do with the iframe? (Custom components are rendered in iframes...)

Can you try [something similar to this test](https://github.com/mlcommons/croissant/blob/475998a89716a8a022817f1126cf3dd1a3526418/editor/cypress/e2e/renameDistribution.cy.js#L25-L28)?

Reply:
Ah no, it's on the other panel.

Reply:
OK for me to not test it if it takes too much time to figure out why it doesn't work.

Reply:
I did it like in that test, it doesn't have a problem to click on the created file, and also correctly sees the `SHA256*` field with the correct value.  But the `Name*` always equals "MyDataset" instead of "test.csv".  I'll try to fix it and if not, leave with only the `SHA256` which still tests that this is correctly created.

Reply:
I'll just leave it as is with testing only the `SHA256*`, we can modify the test later when I add  one for `FileSet` - also takes some time to figure out how to select a value from the `streamlit` dropdown...